C/plugins/south/dht11/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 2.4.0)
```

Comment:
we have this 2.4.0 at some places and 2.8.8 at some; not sure if this should be a const at common place? so that we can use that in every (any) CMakeLists.txt.

Reply:
This is the oldest cmake version having all the features that the cmake script needs. Since the various cmake scripts in our code base use different cmake features, so I guess it is Ok to have different values in various cmake files.